planty/planty/utils.py

```diff
@@ -30,3 +31,9 @@ def save_cropped_image(image_path, cropped_dim):
         img_cropped.thumbnail((cropped_dim, cropped_dim))
 
     img_cropped.save(image_path)
+
+
+def paginate(request, posts, number_posts):
```

Comment:
NIT: could indicate this is a more general function by renaming `posts` to `instances` and `number_posts` to `number_instances`